packages/perseus-editor/src/widgets/plotter-editor.tsx

```diff
@@ -146,52 +327,34 @@ class PlotterEditor extends React.Component<Props, State> {
             <div className="perseus-widget-plotter-editor">
                 <div>
                     Chart type:{" "}
-                    {_.map(
-                        [BAR, LINE, PIC, HISTOGRAM, DOTPLOT],
-                        function (type) {
-                            return (
-                                <label key={type}>
-                                    <input
-                                        type="radio"
-                                        name="chart-type"
-                                        // @ts-expect-error [FEI-5003] - TS2683 - 'this' implicitly has type 'any' because it does not have a type annotation.
-                                        checked={this.props.type === type}
-                                        onChange={_.partial(
-                                            // @ts-expect-error [FEI-5003] - TS2683 - 'this' implicitly has type 'any' because it does not have a type annotation.
-                                            this.changeType,
-                                            type,
-                                        )}
-                                    />
-                                    {type}
-                                </label>
-                            );
-                        },
-                        this,
-                    )}
+                    {[BAR, LINE, PIC, HISTOGRAM, DOTPLOT].map((type) => {
```

Comment:
I wonder if we should make a Typescript enum out of this array.  Or rather, use the enum style that Ned [posted](https://khanacademy.slack.com/archives/C028T2HDX3P/p1684430521976579) about a while back:

```js
export const plotTypes = [BAR, LINE, PIC, HISTOGRAM, DOTPLOT] as const;
export type PlotType = typeof plotTypes[number];
```